.github/workflows/pytests.yml

```diff
@@ -0,0 +1,45 @@
+name: Unit and Integration tests with pytest
+
+on:
+  pull_request:
+    types: [ assigned, opened, synchronize, reopened ]
+    #    paths: [ 'services/py_api/**' ]
```

Comment:
Probably uncomment and make it detect only *.py code, toml and lockfile changes

Reply:
@asynchroza Fixed in [9cc55f2](https://github.com/AUBGTheHUB/monolith/pull/714/commits/9cc55f22a6ea9470d7b4ccc18d592f7c9f2a6ed9)